extensions/packageManager/src/pages/Library.tsx

```diff
@@ -603,7 +603,7 @@ const Library: React.FC = () => {
           </Link>
         </p>
       </div>
-      <Stack horizontal verticalFill styles={{ root: { borderTop: '1px solid #CCC' } }}>
+      <Stack horizontal verticalFill styles={{ root: { borderTop: '1px solid #CCC', height: 'calc(100% - 150px)' } }}>
```

Comment:
@pavolum can we indicate what the 150px is for using a variable?

Reply:
Okay I have added better naming for the styling.